AsyncDisplayKitTests/ASTextKitCoreTextAdditionsTests.m

```diff
@@ -44,5 +49,21 @@ - (void)testNoAttributeCleansing
   XCTAssertTrue([testString isEqualToAttributedString:actualCleansedString], @"Expected the output string %@ to be the same as the input %@ if there are no core text attributes", actualCleansedString, testString);
 }
 
+- (void)testNSParagraphStyleNoCleansing
+{
+  NSMutableParagraphStyle *paragraphStyle = [[NSMutableParagraphStyle alloc] init];
+  paragraphStyle.lineSpacing = 10.0;
+
+  //NSUnderlineStyleAttributeName flags the unsupported CT attribute check
```

Comment:
Space after // - also should go ahead and link to this issue or give slightly more description on the expected behavior, as it currently is unclear why we'd want an unsupported attribute to make it through